configs/components/rubygem-puppet.rb

```diff
@@ -4,8 +4,8 @@
   pkg.version version
 
   case version
-  when '7.7.0'
-    pkg.md5sum '81cfcab6f6998bf7cfbcf0c91a72a469'
+  when '7.10.0'
+    pkg.md5sum '801e1945b1c483d1d5a4cb9b1caf7578'
   when '6.23.0'
```

Comment:
It would probably be good to get on latest 6.x puppet for PE.

Reply:
d-d-d-done